code/datums/components/scope.dm

```diff
@@ -29,20 +29,20 @@
 	))
 
 /datum/component/scope/process(delta_time)
-	if(!tracker.marksman.client)
-		stop_zooming(tracker.marksman)
+	if(!tracker.owner.client)
```

Comment:
can we cache this to some proc-local variable so we don't have to keep accessing two different things in `process()`? if you're scared of caching the client due to disconnect/BYOND memes, that's fine, but at least cache `tracker.owner`. will make it easier to read for the rest of the proc imo